app/components/styleguides/button.js

```diff
@@ -0,0 +1,82 @@
+import Component from '@ember/component';
+import abstractMethod from 'travis/utils/abstract-method';
+import { computed } from '@ember/object';
+
+const DEFAULT_SHADES = {
```

Comment:
Originally my intent was for a button to have multiple shades available for the variant to use for various states, rather than a single shade. But I think it's too complicated, so yeah, this sounds good to me. And, if we pass in `shade` as a number, maybe it's not necessary to have this dictionary at all, we can just have `DEFAULT_SHADE = 400`.